www-cronrobot/app/javascript/stylesheets/application.scss

```diff
@@ -1,5 +1,534 @@
-@import './bootswatch-variables.scss';
-@import './bootstrap.css';
-@import './bootswatch-core.scss';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap');
 
-@import './font-awesome.scss';
\ No newline at end of file
+html {
```

Comment:
A favicon is missing (small icon in the browser)

Reply:
Done 👍